packages/core/src/components/panel-stack/panelView.tsx

```diff
@@ -42,6 +42,11 @@ export interface IPanelViewProps {
 
     /** Whether to show the header with the "back" button. */
     showHeader: boolean;
+
+    /**
+     * HTML title to be passed to the <Text> component
+     */
+    textTitle?: string;
```

Comment:
I think this should be a property of `IPanel`, as `htmlTitle`. then you can render it in the back button as well (`maybeRenderBack()`)